lib/ansible/module_utils/network/vyos/utils/utils.py

```diff
@@ -11,9 +11,10 @@
 
 
 def search_obj_in_list(name, lst, key='name'):
```

Comment:
This method is already defined in network/common/utils.py. Could you remove this from vyos utils and reuse the common one?